config/flags_payments.go

```diff
@@ -79,11 +73,23 @@ var (
 		Usage:  "The duration we'll wait before giving up trying to fetch new balance.",
 		Hidden: true,
 	}
-	// FlagPaymentsZeroStakeUnsettledAmount determines the minimum amount of myst required before auto settling is triggered if zero stake is used.
-	FlagPaymentsZeroStakeUnsettledAmount = cli.Float64Flag{
-		Name:  "payments.zero-stake-unsettled-amount",
+	// FlagPaymentsAutoPromiseSettleMinAmount represents the minimum amount of myst that we will settle automatically.
+	FlagPaymentsAutoPromiseSettleMinAmount = cli.Float64Flag{
+		Name:  "payments.settle.min-amount",
```

Comment:
This will break everything related to the config and this param. Which is old config and probably UI.